blog/2024-12-05-november-releases.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "November 2024 Releases"
+slug: "2024-12-05-november-releases"
+authors: [kenodegard]
+tags: [announcement, conda, conda-build, conda-libmamba-solver]
+description: |
+  conda 24.11.0, conda-build 24.11.2, and conda-libmamba-solver 24.11.1 have been released! 🎉
+image: img/blog/2024-12-05-november-releases/banner.png
+---
+
+The November 2024 releases included updates to conda, conda-build, and conda-libmamba-solver! 🎉 All of these have been released to both `main` and `conda-forge` channels.
+
+<!-- truncate -->
+
+## Changes in conda [24.11.0](https://github.com/conda/conda/releases/tag/24.11.0)
+
+To update `conda` to the latest version, run:
+
+```bash
+conda install -n base conda=24.11.0
+```
+
+### ✨ What's New? ✨
+
+- Adds a new plugin hook for reporter backends.
+- Add support for CEP-17 that allows specifying the path to the site-packages directory of the the Python package via the `repodata.json`.
```

Comment:
Add / Adds tense inconsistency

Reply:
+1 on this. I believe that we usually go with "add" over "adds" (or the equivalent).